common/src/main/java/io/pravega/schemaregistry/common/HashUtil.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.schemaregistry.common;
+
+import java.math.BigInteger;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public class HashUtil {
+    private static final String SHA_256 = "SHA-256";
```

Comment:
This serves no purpose as it is used once and only a single algorithm is used.

Reply:
Still, keeping string constants as a final static is always better.